src/services/Api/contacts.ts

```diff
@@ -170,3 +170,79 @@ export async function createContacts(
   });
   return contactsData;
 }
+
+export async function createVolunteerContact(
+  token: string,
+  org_id: number,
+  user_id: number,
+  contact: VolunteerContact,
+): Promise<VolunteerContact> {
+  const requestOptions: RequestInit = {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${token}`,
+    },
+    body: JSON.stringify({
+      first_name: contact.first_name,
+      middle_name: contact.middle_name,
+      last_name: contact.last_name,
+      inmate_number: contact.inmate_number,
+      facility_name: contact.facility_name,
+      facility_address: contact.facility_address,
+      facility_city: contact.facility_city,
+      facility_state: contact.facility_state,
+      facility_postal: contact.facility_postal,
+      relationship: contact.relationship,
+      org_id: org_id,
+      s3_image_url: contact.profile_img_path,
+    }),
+  };
+  const response = await fetch(
+    url.resolve(API_URL, 'contact/' + user_id),
+    requestOptions,
+  );
+  const body = await response.json();
+  if (body.status === 'ERROR') {
+    throw body;
+  }
+  interface c {
```

Comment:
so imo this is actually a bad practise / technical debt that we accumulated from the first release.

On letters mobile we declare these API interfaces as RawObject to make it clear that data is unprocessed. 

I'd probably call this RawVolunteerContact and add to our type file instead of declaring here

Reply:
Moved to that file!